third_party/llvm/llvm.bzl

```diff
@@ -85,6 +85,26 @@ def cmake_var_string(cmake_vars):
   return " ".join([_quote("{}={}".format(k, str(v)))
                    for (k, v) in cmake_vars.items()])
 
+def expand_isl_cmake_vars(name, src, dst, cmake_vars):
```

Comment:
Is this the same as `expand_cmake_vars`?  If not, please add a comment on how they're different; otherwise please just use `expand_cmake_vars`.

Reply:
The parsing tool used would be `expand_isl_cmake_vars` in this function (which has a different regex for parsing isl files as mentioned above).